### DISCUSSION TITLE
Can an admin protect the `humble` branch?

### DISCUSSION THREAD

**Discussion by sea-bass** [RobotWebTools/rosbridge_suite#936] [General]
Due to the changes described in https://github.com/RobotWebTools/rosbridge_suite/pull/932, I had to create a new `humble` branch that diverges from `ros2`.

However, this branch doesn't seem to be protected by reviews/PRs and I don't have the power to do that. Anyone able to set this up with the same protection rules as `ros2`? Thanks!

**Comment by sea-bass** [comment]
@MatthijsBurgh also pointed out "We need to update the branch protection to remove the required humble check."

**Comment by MatthijsBurgh** [comment]
This should be fixed
